video/hexload.h

```diff
@@ -89,22 +101,53 @@ void VDUStreamProcessor::vdu_sys_hexload(void) {
 				ez80checksum += readByte_b();		// get feedback from ez80 - a 2s complement to the sum of all received bytes, total 0 if no errorcount      
 				ihexchecksum += getIHexByte();		// finalize checksum with actual checksum byte in record, total 0 if no errorcount
 				if(ihexchecksum || ez80checksum) errorcount++;
-				echo_checksum(ihexchecksum,ez80checksum);
+				if(u >= DEF_U_BYTE) echo_checksum(ihexchecksum,ez80checksum);
+				else printFmt("R");

```

Comment:
bike-shedding comment alert....

I prefer `if` statements (in c-style languages) to always use blocks.  I find code like that much clearer to understand, more maintainable, and thus less error-prone in the longer term.  with an `if` like this, for instance, I have to think harder about it to understand what's going on, and am never _quite_ sure of the rules.

no need to change this.  it's just my preference

at some point though I'll work out adding tooling to this codebase that does automated linting and code style things.  then we'll _all_ just be annoyed instead by the linting tools on a CI complaining about our commits :grin: